packages/styled-system/src/css.ts

```diff
@@ -98,14 +164,14 @@ export const css = (args: StyleObjectOrFn = {}) => (
     const value = config?.transform?.(val, scale) ?? get(scale, val, val)
 
     if (config?.properties) {
-      for (const property of config?.properties) {
+      for (const property of config.properties) {
```

Comment:
as below, the `if` statement already ensures this is truthy - another case ESLint would've caught which is why after this PR and the ColorMode PR have been merged, I'd like to implement proper linting